### ISSUE TITLE
Missing visible focus for elements in cards (eg spaces)

### ISSUE THREAD

**Issue by krallowski** [humhub/humhub#8425]
The cards (Kacheln) eg in the overview of spaces have elements with a missing visible focus on keyboard focus:

- Space Icon and Header (maybe that is already fixed as I did not saw a focus in the humhub community space overview)
- Heading and description part (link to the space)
- Members

### What steps will reproduce the problem?
Go to the spaces overview and tab through the cards

### What is the expected result?
I get a visible cue for the focused elements

### What do you get instead?
No visible focus for the mentioned elements

### Additional info

| Q                | A
| ---------------- | ---
| HumHub version   | 1.18
| PHP version      | 
| Operating system |

> Note: Also provide the full error stacks from `Administration --> Information --> Logging` and errors from your browser console if possible.
> Note: Please use the [community](https://community.humhub.com) site for any questions on customizing HumHub.ay

**Comment by yurabakhtin** [comment]
@luke- PR https://github.com/humhub/humhub/pull/8435
